unity/Holo/Assets/Scripts/STLConverter.cs

```diff
@@ -0,0 +1,178 @@
+﻿using Unity;
+using UnityEngine;
+using UnityEditor;
+using System.Text;
+using System.Collections.Generic;
+using System.IO;
+
+/// A class for importing and converting StL series into an animation
+public class STLSeriesConverter
+{
+    GameObject series_GameObject;
+    
+    [MenuItem("Holo/Convert STL series to a .prefab")]
+    public static void ConvertSTL()
+    {
+        STLSeriesConverter stlSeriesConverter = new STLSeriesConverter();
+        STLSeriesImporter stlImporter = new STLSeriesImporter();
+
+        stlSeriesConverter.series_GameObject = stlImporter.Get_GameObject();
+        stlSeriesConverter.series_GameObject.AddComponent<BlendShapeAnimation>();
+
+        stlSeriesConverter.ExportToPrefab();
+    }
+
+    // Exports finished GameObject to a .prefab
+    private void ExportToPrefab()
+    {
+        //string save_path = EditorUtility.SaveFilePanel("Choose a folder where for a .prefab file", "", "", ".prefab");
+        string save_path = @"C: \Users\mit - kuchnowski\repos\holo\unity\Holo\Assets\Resources\test2.prefab";
+        PrefabUtility.SaveAsPrefabAsset(series_GameObject, save_path);
```

Comment:
You should be able to use a relative path, inside project's Assets/, so it works on everyone's machines.